### PR TITLE
user guides: Review user documentation for Organization Basics, Part 2.

### PR DESCRIPTION
**Pull request by ViRu-ThE-ViRuS**
GCI TASK --> https://codein.withgoogle.com/dashboard/task-instances/5935267229728768/

### CODE REVIEW

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part2.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+I did all the listed operations on a local dev instance in MacOS High Sierra, by following all the guides word by word.
+
+### Breakdown:
+#### Import users and channels from Slack
+[Link.](https://zulip.com/help/import-users-and-channels-from-slack)
+The link `Your Account` is pretty much useless and irrelevent. The steps are otherwise easy to follow and appropriate.
+
+#### Restrict new users by email domain
+[Link.](https://zulip.com/help/restrict-user-email-addresses-to-certain-domains)
+The steps for this operation are straight forward and easy to follow. I have no suggestions.
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part2.md]: `straight forward`->`straightforward`

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part2.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+The procedures and user experience for all the above operations åre pretty straightforward.
+The key is then simplicity: we don't want to over complicate the steps for a simple task.
+I did all the listed operations on a local dev instance in MacOS High Sierra, by following all the guides word by word.
+
+### Breakdown:
+#### Import users and channels from Slack
+[Link.](https://zulip.com/help/import-users-and-channels-from-slack)
+The link `Your Account` is pretty much useless and irrelevent. The steps are otherwise easy to follow and appropriate.
+
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part2.md]: `irrelevent`->`irrelevant`

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part2.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+#### Restrict new users by email domain
+[Link.](https://zulip.com/help/restrict-user-email-addresses-to-certain-domains)
+The steps for this operation are straight forward and easy to follow. I have no suggestions.
+
+#### Allow joining without an invitation
+[Link.](https://zulip.com/help/allow-anyone-to-join-without-an-invitation)
+The step 2 has incorrect English. It should be `To limit new members of your organization to those who have been invited, click the  Users need an invitation to join checkbox.` instead of `To limit new members of your organization to those who have been invited to do so, click the  Users need an invitation to join checkbox.`.
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part2.md]: `The step 2`->`Step 2`

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part2.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+### Review User Guides On:
+ - Import users and channels from Slack
+ - Restrict new users by email domain
+ - Allow joining without an invitation
+ - Manage who can send invitations
+ - Set the default language for new users
+
+The procedures and user experience for all the above operations åre pretty straightforward.
+The key is then simplicity: we don't want to over complicate the steps for a simple task.
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part2.md]: `then`->`the`

**Comment by shubhamdhama** [comment]
Some typo and grammatical fixes will make this PR awesome. Though Great content and feedback.

**Comment by ViRu-ThE-ViRuS** [comment]
Done. Thanks again :)

**Comment by shubhamdhama** [comment]
@ViRu-ThE-ViRuS Though not mentioned in Task B, I hope you will take care that line wrap the markdown document to 79 characters so it is easier to read in diff.

**Pull request merged**